packages/@lwc/engine-server/src/apis/render-component.ts

```diff
@@ -44,8 +45,8 @@ export function renderComponent(
         );
     }
 
-    const element = createElement(tagName);
-    createVM(element, Ctor, {
+    const element = renderer.createElement(tagName);
+    createVM(element, Ctor, renderer as unknown as RendererAPI, {
```

Comment:
Works even without typing it to `unknown`
```suggestion
    createVM(element, Ctor, renderer as RendererAPI, {
```